qutebrowser/mainwindow/mainwindow.py

```diff
@@ -186,7 +187,7 @@ def __init__(self, *, private, geometry=None, parent=None):
             private = True
         else:
             private = bool(private)
-        self._private = private
+        self.private = private
```

Comment:
nitpick: Please also remove the leading underline in the class docstring (where the `private` attribute is documented).

Reply:
:ok_hand: